tendrl/integrations/gluster/utils.py

```diff
@@ -0,0 +1,22 @@
+def update_volume_alert_count(alert, existing_alert):
```

Comment:
You probably have https://github.com/Tendrl/node-agent/pull/586/files#diff-fc525d4bb65f3b413979c834c12b06d0R95 so is this required?

Reply:
This function is for calculate volume alert count, so i implemented here , this https://github.com/Tendrl/node-agent/pull/586/files#diff-fc525d4bb65f3b413979c834c12b06d0R95 will call
update_volume_alert_count() to calculate volume alert count

Reply:
But why is this required?
I see everythin here except counter_obj creation is there in https://github.com/Tendrl/node-agent/pull/586/files#diff-fc525d4bb65f3b413979c834c12b06d0R95 can you move that counter_obj creation there?

Reply:
modified this logic